src/mainwindow/mainwindow.h

```diff
@@ -33,6 +33,8 @@ class MainWindow : public QMainWindow
 
     void setupMainMenu();
 
+    void SaveState();
+    void RestoreState();
 private slots:
```

Comment:
Provide some comments to methods saveState / restoreState 
They are not obvious for the others

Reply:
done